### ISSUE TITLE
Service generation issue with symbol = as parameter

### ISSUE THREAD

**Issue by Erim32** [cyclosproject/ng-openapi-gen#95]
Great work and very useful project, I just discovered an issue with get parameters.
I hope that can help you to improve your project.

OpenApi v3 extract of Get => parameters :

```
{
            "name": "=",
            "in": "query",
            "required": false,
            "description": "Get entries that matches exactly your input",
            "schema": {
              "type": "string"
            },
            "deprecated": false
 },
```
 
Extract of the generated service :
```
  methodName$Response(params?: {
    limit?: number;
    ?: string;
    ne?: string;
   ...
```
Issue :
Generate ";" and not "," also the "=" cant be convert as variable name.

More informations:
https://stackoverflow.com/questions/1661197/what-characters-are-valid-for-javascript-variable-names

**Comment by luisfpg** [comment]
The fix would be similar to #93: use the name as string when there are invalid characters.
For example: 
```
 methodName$Response(params?: {
    limit?: number;
    '='?: string;
    ne?: string;
   ...
```

**Comment by Erim32** [comment]
Thank you, looks work ( I can't test now ), do you think that will be possible for you to add a fix to support manage restricted words like this ?


 ```
   /**
     * Get entries that matches exactly your input
     */
    '='?: string;
    rb.query('=', params["="]);

```
